src/storyshots.test.js

```diff
@@ -34,14 +53,21 @@ initStoryshots({
   test: ({story, context, done}) => {
     const converter = new Stories2SnapsConverter();
     const snapshotFilename = converter.getSnapshotFileName(context);
-    const tree = TestRenderer.create(story.render(), {createNodeMock});
-
-    // Because observables are async, execute snapshot tests on next event loop cycle
-    setTimeout(() => {
-      if (snapshotFilename) {
-        expect(tree.toJSON()).toMatchSpecificSnapshot(snapshotFilename);
-      }
-      done();
-    }, 0);
+    let tree;
+
+    global.navigator.mediaDevices = mediaDevicesMock;
```

Comment:
Maybe we could use Enzyme or render from React Testing Library.
https://www.npmjs.com/package/@storybook/addon-storyshots
Check the addon-storyshots documentation, this is what they recommend:
"By design, react-test-renderer doesn't use a browser environment or JSDOM. Because of this difference, some stories might render in your browser, but not in Storyshots. If you encounter this problem, you may want to switch for an higher level renderer such as mount from Enzyme or render from React Testing Library."